msbuild/Xamarin.MacDev.Tasks/Tasks/FilterStaticFrameworks.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Build.Tasks;
+using Microsoft.Build.Framework;
+using Xamarin.Messaging.Build.Client;
+
+namespace Xamarin.MacDev.Tasks {
+	public class FilterStaticFrameworks : FilterStaticFrameworksTaskBase, ITaskCallback {
+		public override bool Execute ()
+		{
+			if (ShouldExecuteRemotely ())
+				return new TaskRunner (SessionId, BuildEngine4).RunAsync (this).Result;
+
+			return base.Execute ();
+		}
+
+		public bool ShouldCopyToBuildServer (ITaskItem item) => false;
```

Comment:
This should return true, so the input files are copied over to the Mac. Sorry I didn't mention it on my previous comment :)

Reply:
@emaf Fixed!